conf/ocsci/fdf_pre_ga_deployment.yaml

```diff
@@ -0,0 +1,12 @@
+---
+DEPLOYMENT:
+  fusion_deployment: true
```

Comment:
I see none of the config values are documented in:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md

Can we get them documented please as well?

Reply:
All should be documented now.